### DISCUSSION TITLE
Quick switch android 13 support

### DISCUSSION THREAD

**Discussion by lordofhype** [LawnchairLauncher/lawnchair#3466] [Q&A] [answered]
Greetings. Is it possible to add quick switch support for lawnchair 9.1 alpha 3? Exactly this version, cuz it has the most flexible customization like blur for dock, correct applying icon packs and etc.
Maybe some magisk modules... I found that for version 12.1.0, but this version still has problems with icon packs and poor customization 
Thanks in advance

**Comment by SuperDragonXD** [comment, marked as answer]
Hello! I know that this reply is a bit late, but as seen [in our FAQ](https://lawnchair.app/faq/#what-happened-to-lawnchair-legacy), we no longer support Lawnchair 9.1 (Legacy). However, Lawnchair 15 does support QuickSwitch from Android 10 to Android 15 QPR0.
